scripts/superdesk-monitoring/monitoring.js

```diff
@@ -122,13 +122,13 @@
                             {term: {'task.last_authoring_desk': desk_id}},
                             {and: [
                                 {term: {'task.desk': desk_id}},
-                                {terms: {state: ['scheduled', 'published', 'corrected', 'killed']}}
```

Comment:
Hi @ioanpocol, I checked with Lasky and we need to keep `scheduled` state in desk output. There's no problem for us in having a separate section called `scheduledDeskOutput` for `scheduled` items though. So maybe we can remove `scheduled` items from desk output only if `scheduledDeskOutput` view is enabled. If you think it is hard and complicated any other solution from you is welcome as well.